backend/app/services/cassandra_service.py

```
@@ -0,0 +1,740 @@
+
+            # Update analytics (async, non-blocking)
+            asyncio.create_task(
+                self._update_analytics(
+                    client_account_id,
+                    user_id,
+                    search_query,
+                    results_count,
+                    execution_time_ms,
+                    source,
+                )
+            )
```

Ruff (0.14.10)

Store a reference to the return value of `asyncio.create_task`

(RUF006)